octoprint_discordremote/discord.py

```diff
@@ -432,4 +466,4 @@ def check_errors(self):
                 self.status_callback(connected="disconnected")
 
     def log_safe(self, message):
-        return message.replace(self.bot_token, "[bot_token]").replace(self.channel_id, "[channel_id]")
+        return message.replace(self.bot_token, "[bot_token]").replace(self.channels_id, "[channels_id]")
```

Comment:
This is not correct, you need to search for each channel ID, and replace them one by one.
```suggestion
        clean_message = message.replace(self.bot_token, "[bot_token]")
        for channel_id in self.channels_id:
            clean_message = clean_message.replace(channel_id, "[channel_id]")
        return clean_message
```